hi-ml-histopathology/src/histopathology/utils/output_utils.py

```diff
@@ -390,14 +395,16 @@ def _save_outputs(self, epoch_results: EpochResultsType, outputs_dir: Path) -> N
 
         save_outputs_csv(results, outputs_dir)
 
-        print("Selecting tiles ...")
-        selected_slide_ids = save_top_and_bottom_tiles(results, n_classes=self.n_classes, figures_dir=figures_dir)
+        if self.save_tiles:
+            logging.info("Selecting tiles ...")
+            selected_slide_ids = save_top_and_bottom_tiles(results, n_classes=self.n_classes, figures_dir=figures_dir)
 
-        if self.slides_dataset is not None:
-            save_slide_thumbnails_and_heatmaps(results, selected_slide_ids, tile_size=self.tile_size, level=self.level,
-                                               slides_dataset=self.slides_dataset, figures_dir=figures_dir)
+            if self.slides_dataset is not None:
+                save_slide_thumbnails_and_heatmaps(results, selected_slide_ids, tile_size=self.tile_size,
+                                                   level=self.level, slides_dataset=self.slides_dataset,
+                                                   figures_dir=figures_dir)
 
-        save_scores_histogram(results, figures_dir=figures_dir)
+            save_scores_histogram(results, figures_dir=figures_dir)
```

Comment:
oh my bad. I'll put it back! I first was planning on asking people who wants to run the slides pipeline to comment these lines but then I added this parameter to avoid having to comment/uncomment code snippets. I indent it by accident.